rest_auth_toolkit/views.py

```diff
@@ -51,12 +51,15 @@ def post(self, request):
         user = deserializer.save()
 
         if self.email_confirmation_class is None:
```

Comment:
I think line 53 to 56 should go after the `if get_setting('email_confirmation_send_email', True):`
That way there would be no need to define the email_confirmation_class if the email_confirmation_send_email is set to False.

Other option would be to let is as is and let the programer decide how he wants to validate the emails.

Reply:
I think the idea was that you always want a record in your DB (using your email confirmation class), but not always want to send actual email messages.

Reply:
Problem is known #38 and has a PR #44